test/new_relic/slack_notifications_tests/slack_notifier_tests.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require_relative '../../../.github/workflows/scripts/slack_notifications/slack_notifier'
+
+class SlackNotifications < Minitest::Test
+  def test_send_slack_message_zero_args
+    SlackNotifier.stub(:sleep, nil) do
```

Comment:
Does this stub get called when the test runs?

Reply:
Yes. Like what we did before, I changed `sleep(1)` to `sleep(100)`. Didn't slow down.

Reply:
If you temporarily change `sleep(1)` to `sleep(100)` in the code and then change `SlackNotifier.stub(:sleep)` into `SlackNotifier.stub(:doze)` here in this one test, does the test slow down by 100 seconds?

Reply:
It's working! The interpreter doesn't like `doze` since its not a real method. But I added a `puts` and stubbed out `puts` instead of `sleep` and the test took a long time to run.

Reply:
oh and importantly in this^ test, my `puts` never showed up when it was stubbed

Reply:
[Done, thank!](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/commits/7ade7858803276a1efd309ab396e6f00a5bf4a42)!